cMake/FindShiboken2.cmake

```diff
@@ -0,0 +1,29 @@
+# - Find Shiboken2 installation
+#
+# The following variable are set:
+#   Shiboken2_FOUND   - Shiboken2 was found
+#   SHIBOKEN2_FOUND   - Shiboken2 was found
+#   Shiboken2_DIR     - The Shiboken2 libraries dir
+#   SHIBOKEN2_LIBRARY - The Shiboken2 libraries
+
+
+execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} -c "import shiboken2;import os;print(os.path.dirname(shiboken2.__file__), end='')"
+    RESULT_VARIABLE FAILURE
+    OUTPUT_VARIABLE PRINT_OUTPUT
+)
+
+set(SHIBOKEN2_FOUND FALSE)
+if(NOT FAILURE)
+    set(SHIBOKEN2_FOUND TRUE)
+    set(SHIBOKEN2_LIBRARY ${PRINT_OUTPUT} CACHE PATH "The location of the Shiboken2 library")
```

Comment:
I am really not sure if this is really what we want here. I guess we need "libshiboken" instead of the python library. Have a look at the filestructure of the conda pyside2-package:

```
pyside2-5.9.0a1-py37h4dc837a_0/
├── lib
│   ├── cmake
│   │   ├── PySide2-5.9.0
│   │   │   ├── PySide2Config.cmake
│   │   │   ├── PySide2Config.cpython-37m-x86_64-linux-gnu.cmake
│   │   │   └── PySide2ConfigVersion.cmake
│   │   └── Shiboken2-5.9.0
│   │       ├── Shiboken2Config.cmake
│   │       ├── Shiboken2Config.cpython-37m-x86_64-linux-gnu.cmake
│   │       └── Shiboken2ConfigVersion.cmake
│   ├── libpyside2.cpython-37m-x86_64-linux-gnu.so.5.9.0
│   ├── libpysidetest.so
│   ├── libshiboken2.cpython-37m-x86_64-linux-gnu.so.5.9.0
│   └── python3.7
│       └── site-packages
│           ├── PySide2
│           │   ├── _config.py
│           │   ├── _git_pyside_version.py
│           │   ├── __init__.py
│           │   ├── __pycache__
│           │   │   ├── _config.cpython-37.pyc
│           │   │   ├── _git_pyside_version.cpython-37.pyc
│           │   │   └── __init__.cpython-37.pyc
│           │   └── ...
│           ├── pyside2uic
│           │   └── ...
│           └── shiboken2.cpython-37m-x86_64-linux-gnu.so
└── ...
```
The path we derive by importing the shiboken2 via python will be something like `.../lib/pythonx.y/site-packages/shiboken2*.so` but actually we need `.../lib/libshiboken2*.so`. Maybe there is a method to derive this path via the shiboken python module, but I am not sure. I guess there will be a similar problem for pyside2 / libpyside2...